src/config/config.ts

```diff
@@ -0,0 +1,11 @@
+import { Sequelize } from 'sequelize-typescript';
+
+const sequelize = new Sequelize({
+    database: 'foodies-haven',
+    username: 'root',
+    password: 'password',
+    host: '127.0.0.1',
+    dialect: 'mysql'
+});
+
+export default sequelize;
```

Comment:
Please add a new line after this
```suggestion
export default sequelize;

```